apps/src/templates/instructions/TopInstructionsCSF.jsx

```diff
@@ -183,29 +169,36 @@ var TopInstructions = React.createClass({
     const renderedMarkdown = processMarkdown(this.props.collapsed ?
       this.props.shortInstructions : this.props.longInstructions);
 
+    // TODO - the colWidth numbers are kind of magic/arbitrary right now (it's the
+    // amount needed for the collapser button and the hint icon), and could likely
+    // become more dynamic - or at least more well documented - in the future
     return (
       <div style={mainStyle} className="editor-column">
-        <div>
-          <div style={styles.body}>
-            {this.props.longInstructions && <CollapserButton
-                style={[styles.collapserButton, this.props.isRtl && styles.collapserButtonRtl]}
-                isRtl={this.props.isRtl}
-                collapsed={this.props.collapsed}
-                onClick={this.handleClickCollapser}/>
-            }
-            {<Instructions
-                ref="instructions"
-                renderedMarkdown={renderedMarkdown}
-                onResize={this.props.onResize}
-                inTopPane
-            />
-            }
+        <ThreeColumns
+            style={styles.body}
+            leftColWidth={90}
+            rightColWidth={90}
+            height={this.props.height - resizerHeight}
+        >
+          <div id="bubble" id="prompt-icon-cell">
+            <PromptIcon src={this.props.smallStaticAvatar}/>
```

Comment:
This has some potential to be problematic since we have code modifying this elsewhere (i.e. add a click handler to bubble and potentially insert a lightbulb into the dom if we have authored hints). I think I can just make this a ProtectedStatefulDiv.

I think that the next PR will have redux know whether or not we have authored hints, such that we can at least avoid adding DOM elements from outside of react.